apps/researcher/src/app/[locale]/(objects)/objects/[id]/user-enrichment-form.tsx

```diff
@@ -161,7 +223,11 @@ function Form({
   );
 }
 
-export function UserEnricherForm({slideOutId, identifier, objectId}: Props) {
+export function UserEnricherForm({
```

Comment:
Perhaps rename to `UserEnrichmentForm`, to be in sync with the name of the file?